exchange/gdpr_test.go

```diff
@@ -0,0 +1,119 @@
+package exchange
+
+import (
+	"testing"
+
+	"github.com/mxmCherry/openrtb"
+)
+
+func TestExtractGDPRFound(t *testing.T) {
```

Comment:
https://github.com/prebid/prebid-server/pull/658#discussion_r212363297 reminded me...

It sounds like we both think the `exchange` code can be a little hard to read right now. I would recommend making JSON-style tests for these, so that the tests will still work if/when the code gets too messy and needs refactoring.

I wouldn't block the merge for it, though... it'll just make future refactors of this module easier.

Reply:
JSON-style tests will require a lot of extra work to test everything that needs testing, so will save that for another day.